### PR TITLE
Sensor dev

### PR DESCRIPTION
**Pull request by christomaszewski**
Github cleanup - merge into master prior to branching for new feature development.

### CODE REVIEW

File: firmware/firmware.ino [hunk 1]
```diff
@@ -214,6 +215,42 @@ void handleCommand(const char *buffer)
...
+      } else {
+        reportError("Unknown compensation entry.", buffer);
+        return;
+      }
+
+      //No sensors to compensate
+      if (entry_object == NULL){
+        return;
+      }
+    }*/
```
Comment (psigen) [inline firmware/firmware.ino]: @christomaszewski: can we remove this unused comment?

File: firmware/firmware.ino [hunk 2]
```diff
@@ -383,13 +421,48 @@ void loop()
   input_buffer[bytes_read] = '\0';
   
   // Copy incoming message to debug console.
-  Serial.print("<- ");
-  Serial.println(input_buffer);
-    
+  //Serial.print("<- ");
+  //Serial.println(input_buffer);
+
```
Comment (psigen) [inline firmware/firmware.ino]: Let's just remove this if we are not using it.

File: firmware/firmware.ino [hunk 3]
```diff
@@ -292,7 +329,8 @@ void setup()
   
   // Create secondary tasks for system.
   Scheduler.startLoop(motorUpdateLoop);
-  Scheduler.startLoop(serialConsoleLoop);
+  //Scheduler.startLoop(serialConsoleLoop);
```
Comment (psigen) [inline firmware/firmware.ino]: Was there a reason to remove this? Was it causing a delay?

File: firmware/firmware.ino [hunk 4]
```diff
@@ -383,13 +421,48 @@ void loop()
...
+  
+  //assumes es2 on port 3 and Do probe on port 2
+  double newTemp = ((platypus::ES2*)platypus::sensors[3])->getLastTemp();
+  if (newTemp != lastTemp){
+    //send updated temp to do probe
+    ((platypus::AtlasSensor*)platypus::sensors[2])->setTemp(newTemp);
+    lastTemp = newTemp;
+    Serial.println(newTemp);
+  }
+}*/
```
Comment (psigen) [inline firmware/firmware.ino]: If we use this loop sometimes, we should uncomment it, but not call it.
If we don't use this loop, we should just remove it.

File: firmware/firmware.ino (hunk 2 above)
Comment (christomaszewski) [inline firmware/firmware.ino]: I want to make this enabled or disabled based on the debug level

File: firmware/firmware.ino (hunk 3 above)
Comment (christomaszewski) [inline firmware/firmware.ino]: Unknown, we disabled a bunch of stuff once we noticed a delay in Qatar. Maybe make it enabled/disabled based on debug level?

File: firmware/firmware.ino (hunk 1 above)
Comment (christomaszewski) [inline firmware/firmware.ino]: For Sensor compensation. I'll remove it from master for now, then add it back in when I start the Feature-SensorCompensation branch

**Pull request closed without merging**